src/CalcManager/Ratpack/CalcErr.h

```diff
@@ -41,45 +41,45 @@ typedef int32_t ResultCode;
 // CALC_E_DIVIDEBYZERO
 //
 // The current operation would require a divide by zero to complete
-#define CALC_E_DIVIDEBYZERO ((uint32_t)0x80000000)
 
+static constexpr uint32_t CALC_E_DIVIDEBYZERO = (uint32_t)0x80000000;
```

Comment:
Looks like the line feed separation got mixed up for this one.  The const should be connected to the comment like the others.